Purchases/Purchasing/StoreKit2/ProductsFetcherSK1.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ProductsFetcherSK1.swift
+//
+//  Created by Andrés Boedo on 7/26/21.
+
+import Foundation
+import StoreKit
+
+class ProductsFetcherSK1: NSObject {
```

Comment:
this is what used to be `ProductsManager`, but now that class is in charge of deciding between sk1 or sk2, and calling the fetcher for each